prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -642,6 +642,95 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             .into(Setting::class.java)
     }
 
+    /** data returned by fetchSettingRevisionHistory. Only used to shape json response. **/
+    data class SettingsHistoryData(
```

Comment:
This data class looks out of place here.  Find where the other classes are and put them in the same area and level.

Reply:
Will do.

Note: This location was intentional but maybe it's not a standard java style. If I could have scoped it to just the function doing the json serialization I would, but Kotlin/Java doesn't seem to support defining data classes within function scopes. It should definitely not be in the global scope since it's ONLY use is to serialize the result of the query.

Reply:
OK. I've looked around and cannot figure out where the best place for this is. All the other types in the DatabaseAccess file seem to come out of pojo's auto generating types from schemas.

I see a similar style used with `AuthClaims` where the data class is defined right next to the function that uses it.

I'm not asserting some strong style opinion, I literally just don't know the best place for it so I can resolve this issue.

Any suggestions?